src/sql/azdata.d.ts

```diff
@@ -2524,42 +2524,38 @@ declare module 'azdata' {
 		 * Azure Key Vault
 		 */
 		AzureKeyVault = 3,
-		/**
-		 * Azure AD Graph
-		 */
-		Graph = 4,
 		/**
 		 * Microsoft Resource Management
 		 */
-		MicrosoftResourceManagement = 5,
+		MicrosoftResourceManagement = 4,
```

Comment:
I would also suggest looking at any APIs we have that take in these values and making sure they gracefully handle a scenario where an extension calls them with an invalid (in this case deprecated) value. We can't control what extensions are doing so trying to make it so at least a clear error is thrown instead of a null-ref or something would be great.